### PR TITLE
ci: fix debug runner step

### PR DESCRIPTION
**Pull request by belgattitude**
Fix debug runner step and output examples/simple output.  

Why needed ? 

- test-e2e works on ci, not on my local machine. without lock file (choice made before), I'd like to exclude resolutions problems (reproducibility).

In the future I feel we would benefit to commit lock files (or at least for the base package).

See output in https://github.com/i18next/next-i18next/actions/runs/3410769134/jobs/5674130702

image

### CODE REVIEW

**Comment by belgattitude** [comment]
Can be merged :)

**Comment by belgattitude** [comment]
@adrai as I have write access, do I have your permission to merge ? I'll try to be as cautious as possible and ask reviews whenever I have doubts

**Comment by adrai** [comment]
go for it ;-)

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -30,9 +30,13 @@ jobs:
...
 
-      - name: ⓘ Debug runner
+      - name: ⓘ Debug next-i18next installation
         shell: bash
-        run: npx -y envinfo --binaries --npmPackages
+        run: npx envinfo --binaries --npmPackages
+
+      - name: ⓘ Debug example/simple installation
+        shell: bash
+        run: npx envinfo --binaries --npmPackages
```
Comment (kachkaev) [inline .github/workflows/ci.yml:39]: Can `working-directory` be missing here? Not sure if two actions are expected to be identical 🤔

**Pull request merged**